### PR TITLE
Tauros BST Fix

### LINKED ISSUE #3940: Paldean Tauros has wrong base stats
### Description

All breeds of Paldean Tauros are currently using the same TAUROS_MISC_INFO define which contains the base form's stats, but the Paldean form is supposed to have different base stats.

### Version

1.7.1 (Default)

### Upcoming Version

_No response_

### Discord contact info

moo66

### PR DESCRIPTION
**Pull request by kaicardenas2**
Fix the BST of Paldean Tauros, who had the same BST as regular Tauros when it should be slightly different.'

Sorry for making a second PR i had my branches completely messed up idk how that happened

## Issue(s) that this PR fixes
#3940

## **Discord contact info**
kai.cardenas2

### CODE REVIEW

File: src/data/pokemon/species_info/gen_1.h [hunk 1]
```diff
@@ -10178,16 +10178,26 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
...
         .footprint = gMonFootprint_Tauros,                  \
-        .formSpeciesIdTable = sTaurosFormSpeciesIdTable
+        .formSpeciesIdTable = sTaurosFormSpeciesIdTable     \
+#define TAUROS_BASE STATS                                   \
+        .baseHP        = 75,                                \
+        .baseAttack    = 100,                               \
+        .baseDefense   = 95,                                \
+        .baseSpeed     = 110,                               \
+        .baseSpAttack  = 40,                                \
+        .baseSpDefense = 70,                                \
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info/gen_1.h:10188]: This define isn't needed. You can assign the stats to the original Tauros.

**Review by AlexOn1ine — DISMISSED** [review]

File: src/data/pokemon/species_info/gen_1.h [hunk 2]
```diff
@@ -10184,11 +10178,27 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
...
+#define PALDEAN_TAUROS_BASE_STATS                           \
+        .baseHP        = 75,                                \
+        .baseAttack    = 110,                               \
+        .baseDefense   = 105,                               \
+        .baseSpeed     = 100,                               \
+        .baseSpAttack  = 30,                                \
+        .baseSpDefense = 70,                                \
```
[the remark this answers is not in the document]
Comment (kaicardenas2) [reply src/data/pokemon/species_info/gen_1.h:10195]: oh shit i didnt even realize that 😅

**Comment by kaicardenas2** [comment]
done :)

File: src/data/pokemon/species_info/gen_1.h [hunk 3]
```diff
@@ -10184,11 +10178,18 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
...
     [SPECIES_TAUROS] =
     {
         TAUROS_MISC_INFO,
+        .baseHP        = 75,                                \
+        .baseAttack    = 100,                               \
+        .baseDefense   = 95,                                \
+        .baseSpeed     = 110,                               \
+        .baseSpAttack  = 40,                                \
+        .baseSpDefense = 70,    
+        .baseSpDefense = 70,
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info/gen_1.h:10192]: Sorry, you need to remove the last comma here. :sweat_smile:

Comment (kaicardenas2) [reply src/data/pokemon/species_info/gen_1.h:10192]: damn i REALLY thought i had that

sorry for making you review something so simple 24 times😅

File: src/data/pokemon/species_info/gen_1.h [hunk 4]
```diff
@@ -10214,9 +10215,17 @@ const struct SpeciesInfo gSpeciesInfoGen1[] =
     },
 
 #if P_PALDEAN_FORMS
+#define PALDEAN_TAUROS_BASE_STATS                           \
+        .baseHP        = 75,                                \
+        .baseAttack    = 110,                               \
+        .baseDefense   = 105,                               \
+        .baseSpeed     = 100,                               \
+        .baseSpAttack  = 30,                                \
+        .baseSpDefense = 70                                 \
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info/gen_1.h:10224]: I'm sorry again but that's probably on me. The last backslash is not needed.

File: src/data/pokemon/species_info/gen_1.h (hunk 3 above)
Comment (AlexOn1ine) [reply src/data/pokemon/species_info/gen_1.h:10192]: > damn i REALLY thought i had that
> 
> sorry for making you review something so simple 24 times😅

It's fine but please compile before making a commit 😅

Comment (kaicardenas2) [reply src/data/pokemon/species_info/gen_1.h:10192]: sorry im not on my laptop im literally making the changes on github

Comment (AlexOn1ine) [inline src/data/pokemon/species_info/gen_1.h:10192]: Okay, that's also on me. Punished for not checking thoroughly. You don't need the backslashes here. They are needed for macros but not for normal data. 

I hope this is all :skull:

Comment (AlexOn1ine) [reply src/data/pokemon/species_info/gen_1.h:10192]: Please do it on a computer where you can compile. There is no rush :smile:

Comment (kaicardenas2) [reply src/data/pokemon/species_info/gen_1.h:10192]: lmao im so sorry i cant compile first im on my friends laptop

**Comment by AlexOn1ine** [comment]
@kaicardenas2 Can you include a fix for the wrong evo label on Electrode-H once you get to it?
image

**Comment by kaicardenas2** [comment]
> @kaicardenas2 Can you include a fix for the wrong evo label on Electrode-H once you get to it? image

what should the evo label be?

**Comment by Bassoonian** [comment]
> > @kaicardenas2 Can you include a fix for the wrong evo label on Electrode-H once you get to it? image
> 
> what should the evo label be?

`EVO_ITEM` rather than `EVO_LEVEL`

**Comment by kaicardenas2** [comment]
> @kaicardenas2 Can you include a fix for the wrong evo label on Electrode-H once you get to it? image

im done this, but i cant seem to understand why im getting a compiling error

**Comment by AlexOn1ine** [comment]
> > @kaicardenas2 Can you include a fix for the wrong evo label on Electrode-H once you get to it? image
> 
> im done this, but i cant seem to understand why im getting a compiling error (`src/data/pokemon/species_info/gen_1.h:10218:9: error: field name not in record or union initializer 10218 | .baseHP = 75, /`)

You added a backslash it the end of the macro that isn't needed.

**Comment by AlexOn1ine** [comment]
Superseded by #3970

**Pull request closed without merging**